homeassistant/components/yamaha_musiccast/config_flow.py

```diff
@@ -92,7 +92,9 @@ async def async_step_ssdp(
             return self.async_abort(reason="yxc_control_url_missing")
 
         self.serial_number = discovery_info.upnp[ssdp.ATTR_UPNP_SERIAL]
-        self.host = urlparse(discovery_info.ssdp_location or "").hostname or ""
+        self.host = urlparse(
+            discovery_info.ssdp_location
+        ).hostname  # type: ignore[assignment]
```

Comment:
cc @MartinHjelmare is this better?
ref: https://github.com/home-assistant/core/pull/60749#discussion_r760907283

Reply:
Yes, I think that's ok. Please add a comment why we do this, ie why we know the type already.

Reply:
Done

Reply:
Would this be a better solution? We could add to a global helper module.
https://github.com/home-assistant/core/blob/a0176044b916675fc27f7992876e597c679184e0/homeassistant/components/dlna_dmr/config_flow.py#L43-L53

Reply:
Yeah, maybe. It's nice to avoid type ignores. But let's do that separately so others can comment.

Reply:
(see #60794)